cluster/manifests/wiz/connector/secrets.yaml

```diff
@@ -0,0 +1,58 @@
+{{ if eq .Cluster.ConfigItems.wiz_enable_runtime_monitoring_daemonset "true"}}
+---
+# Source: wiz-kubernetes-integration/charts/wiz-kubernetes-connector/templates/secret-connector.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: wiz-connector-connector
+  namespace: "wiz"
+  labels:
+    helm.sh/chart: wiz-kubernetes-connector-3.1.1
+    app.kubernetes.io/name: wiz-kubernetes-connector
+    app.kubernetes.io/instance: wiz-connector
+    app.kubernetes.io/version: "2.5"
+    app.kubernetes.io/managed-by: Helm
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
+    "helm.sh/hook-weight": "-1"
+type: Opaque
+data:
+  connectorData: "e30="
+---
+# Source: wiz-kubernetes-integration/charts/wiz-kubernetes-connector/templates/service-account-cluster-reader.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: wiz-cluster-reader-token
+  namespace: "wiz"
+  labels:
+    helm.sh/chart: wiz-kubernetes-connector-3.1.1
+    app.kubernetes.io/name: wiz-kubernetes-connector
+    app.kubernetes.io/instance: wiz-connector
+    app.kubernetes.io/version: "2.5"
+    app.kubernetes.io/managed-by: Helm
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation
+    kubernetes.io/service-account.name: wiz-cluster-reader
+type: kubernetes.io/service-account-token
+---
+# Source: wiz-sensor/templates/apikeysecret.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: wiz-api-token
+  namespace: wiz
+  labels:
+    
+    helm.sh/chart: wiz-sensor-1.0.4760
+    image/tag: v1
+    app.kubernetes.io/name: wiz-sensor
+    app.kubernetes.io/instance: wiz-sensor
+    app.kubernetes.io/version: "1.0.4708"
+    app.kubernetes.io/managed-by: Helm
+type: Opaque
+stringData:
+  clientId: "{{ .Cluster.ConfigItems.wiz_api_client_id | base64 }}"
+  clientToken: "{{ .Cluster.ConfigItems.wiz_api_client_token | base64 }}"
```

Comment:
When using `stringData`, then `base64` encoding might not be needed, just fyi. https://kubernetes.io/docs/concepts/configuration/secret/#restriction-names-data

Reply:
@vinaythupili now that I think about it, the values are already base64 encoded before the encryption is done. I think we meant to do `base64Decode` here originally, but since the secret can be base64 anyway, we probably don't need this encoding and can just use the `data` field instead of `stringData`.

Reply:
updated to data 👍

Reply:
Do we still need the `base64` encoding, now that we're using the `data` field that's already base64 encoded? (That's what I understand from Zak's comment above)